### DISCUSSION TITLE
RE: Ukraine invaded

### DISCUSSION THREAD

**Discussion by twistingmercury** [pamidur/aspect-injector#180] [General]
@pamidur,

This is heart-breaking for me.  Back in the late fall of 2014 I managed a team of devs (Global Logic) and got to visit them in L'viv.  After a week there, we traveled to Kyiv and toured the city. It wasn't a week later that in that same square where I stood that the demonstration that ultimately ousted Yanukovych (Good riddance!) occurred.  At the time all of my team members talked about how they wanted closer ties with the EU to improve their prosperity. Each also explicitly mentioned they'd readily defend their homeland.

I fell in love with your country, your people. All of my team were very patriotic, and everyone I met was warm and welcoming.  I was, however, afraid of driving, lol!

If this invasion had happened while I was there, I would have gladly taken up arms to defend my Ukrainian brothers and sisters. I will be donating to your homeland's cause.

Long Live Ukraine!  
Remember Snake Island! 
Russia, go f**k yourself!

**Comment by twistingmercury** [comment]
I was compelled to duplicate this on Linkedin as well.

**Comment by pamidur** [comment]
Hi @twistingmercury , thank you for your support. it means a lot!
We will be victorious! Glory to Ukraine!
